services/web/server/config/server-defaults.yaml

```diff
@@ -0,0 +1,36 @@
+version: '1.0'
+app:
+  client_outdir: /usr/src/app/client
+  host: 127.0.0.1
+  log_level: DEBUG
+  port: 8080
+  testing: true
+cs_s4l:
```

Comment:
this is probably not necessary in there... some leftover from the modeler code??

Reply:
For the moment, I leave it as it is (there are some variables defined for that in the current master version). I will cleanup afterwards. I added a note.